front_end/src/app/admin/events/new-family-registration/new-family-registration.component.ts

```diff
@@ -72,7 +72,7 @@ export class NewFamilyRegistrationComponent implements OnInit {
   }
 
   needGradeLevel(child: NewChild): boolean {
-    return moment(child.DateOfBirth).isBefore(moment().startOf('day').subtract(5, 'y'));
+    return moment(child.DateOfBirth).isBefore(moment().startOf('day').subtract(4, 'y'));
```

Comment:
@timmyg This was a request from Dave. He asked that we show the Age/Grade dropdown if the child is 4 and up. Right now, it only shows if they are 5 and up.